### DISCUSSION TITLE
Error loading files on repository "Error 503 Backend is unhealthy"

### DISCUSSION THREAD

**Discussion by LoginAGo** [community/community#11915] [Other Feature Feedback, Questions, & Ideas]
Error 503 Backend is unhealthy
Backend is unhealthy

Guru Mediation:
Details: cache-den8225-DEN 1645546932 3587218648

Varnish cache server

I am trying to load and view images that I have set for my website https://oneenterprisestech.github.io, and the mages will not load. I got the above error, and there appear to be no resolutions (currently) The images that are linked with the style.css  file are not loading either. There is no background, and my website is publicly visible with SEO. I need a fix to this problem as soon as possible, please.

Sorry, I don't know what to categorize this as, as the bug report page seems to be down as well.

UPDATE: It seems the problem is now only the style.css not loading.

UPDATE: The 503 Error seems to be moving between all of the files on my repository.

UPDATE: Background on my website finally loaded, but I doubt it will stay. The error still seems to be going through all of my files and assets.

**Comment by ABJ4403** [comment]
i get the same thing too on https://raw.githubusercontent.com/jerryn70/GoodbyeAds/master/Hosts/GoodbyeAds.txt
but other resources like https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts still works

**Comment by suklav** [comment]
I was also getting the same error for the contents of my git repositories (https://github.com/suklav/) (for both public and private repo s). I waited for some time and then went back and tried to redownload it, then I succeeded.

**Comment by jfullstackdev** [comment]
I'm not very sure what technology did you use and there are also external links being loaded, 

![image](https://user-images.githubusercontent.com/47092464/182392137-23c96f85-e575-4588-8e68-0ac703696412.png)

you look at the Developer Tools, seems that is not GitHub issue

**Comment by msr8** [comment]
Having the same error on multiple of my repos. Started experiencing it lately and am on Firefox (103.0)

**Reply by wizlee** [reply]
Same here, thought it is because I am downloading potentially harmful content
However when it starts to also block word list I started to doubt that...

**Comment by harsh2021shah** [comment]
Same. What is happening

**Comment by sanjmeh** [comment]
Same error, while loading a screenshot here,
https://github.com/rstudio/dygraphs/issues/new
